### PR TITLE
glsl  support

### PR DESCRIPTION
**Pull request by matszczygiel**
Hello, I added glsl language syntax highlighting.
image
I tried to use glslls https://github.com/svenstaro/glsl-language-server as lsp but couldn't get things working.

### CODE REVIEW

File: runtime/queries/glsl/indents.scm [hunk 1]
```diff
@@ -0,0 +1 @@
+; inherits: c
```
Comment (kirawi) [inline runtime/queries/glsl/indents.scm:1]: We use `indents.toml` instead: https://github.com/helix-editor/helix/issues/114

Comment (matszczygiel) [reply runtime/queries/glsl/indents.scm]: @kirawi  I added `indents.toml` based on https://github.com/nvim-treesitter/nvim-treesitter/blob/master/queries/c/indents.scm

**Review by archseer — APPROVED** [review]

**Comment by archseer** [comment]
Looks good! Can you resolve the conflicts? I just merged the Perl grammar.

**Comment by matszczygiel** [comment]
> Looks good! Can you resolve the conflicts? I just merged the Perl grammar.

Done

**Pull request merged**